src/network/connection.cpp

```diff
@@ -219,16 +216,16 @@ BufferedPacket ReliablePacketBuffer::popFirst()
 		throw NotFoundException("Buffer is empty");
 	BufferedPacket p = *m_list.begin();
 	m_list.erase(m_list.begin());
-	--m_list_size;
 
-	if (m_list_size == 0) {
+	if (m_list.empty()) {
 		m_oldest_non_answered_ack = 0;
 	} else {
 		m_oldest_non_answered_ack =
-				readU16(&(*m_list.begin()).data[BASE_HEADER_SIZE+1]);
+				readU16(&m_list.begin()->data[BASE_HEADER_SIZE + 1]);
```

Comment:
Continuation lines needn't be indented by two tabs unless a one-tab indent aligns them with the following lines. (according to @DS-Minetest :P)

Reply:
LINT says 2 tabs for continued lines. However, I cannot find any rule about how many tabs to use for line continuation (only defined for functions).
Nitpicking.

Reply:
It's the code-style _guidelines_ that need fixing, then :D

Reply:
It might indeed be useful if the code style guidelines were more exact at continuation lines.
(In the lua code style guidelines, for example, there are very clear rules: https://dev.minetest.net/Lua_code_style_guidelines#Continuation_lines)

[Kernel style](https://www.kernel.org/doc/html/latest/process/coding-style.html#breaking-long-lines-and-strings) isn't so exact either:
>Descendants are always substantially shorter than the parent and are placed substantially to the right.

In the first example of the [space paragraph](https://dev.minetest.net/Code_style_guidelines#Spaces) of the code style guidelines, there's indentation with one tab. However, there's no rule that clearly requires one to use either option.
This means that both, one tab and two tabs, are acceptable. (Theoretically one could even try to indent with three tabs.)

All in all, this doesn't really matter. The readability doesn't suffer much from the non-existence of such a rule. However, it might make sense to clearly state that one tab or two tabs can be used, to prevent confusion and useless discussions.

Btw. I can be less pedantic, if you want. :slightly_smiling_face: >_<

Reply:
> Btw. I can be less pedantic, if you want. :slightly_smiling_face: >_<

lol no. I was just joking :)